Ginger/Ginger/Drivers/DriversWindows/MobileDriverWindow.xaml

```diff
@@ -31,6 +37,21 @@
             <!--Top buttons-->
             <Frame x:Name="xMainTopFrame" Grid.Row="0" Grid.Column="1" Margin="0,0,0,0" Background="Black"/>
             <UserControls:ucButton x:Name="xRefreshButton"  Grid.Row="0" Grid.Column="1" ButtonType="ImageButton" HorizontalAlignment="Center" VerticalAlignment="Bottom" ButtonFontImageSize="18" ButtonImageType="Refresh" Click="xRefreshButton_Click" ToolTip="Refresh Device Screenshot" Margin="0,0,0,0" Visibility="Collapsed"/>
+            <UserControls:ImageMakerControl x:Name="xRecordingImage" Grid.Row="0" Grid.Column="1" Height="18" Width="18" ImageType="Camera" Foreground="Red" ImageForeground="Red" HorizontalAlignment="Right" VerticalAlignment="Center" SetBorder="False" ToolTip="Recording..." Margin="0,0,60,0" Visibility="Collapsed" />
+            <!--<UserControls:ImageMakerControl x:Name="xRecordingImage" Grid.Row="0" Grid.Column="1" Height="18" Width="18" ImageType="Camera" Foreground="Red" ImageForeground="Red" HorizontalAlignment="Right" VerticalAlignment="Center" SetBorder="False" ToolTip="Recording..." Margin="0,0,60,0" Visibility="Collapsed">
+                <Style>
+                    <Style.Triggers>
```

Comment:
Do cleanup for below, try to make the blinking recording work, if not working let me know and I'll try

Reply:
Done